src/rgw/rgw_op.cc

```diff
@@ -2980,6 +2980,19 @@ int RGWListBuckets::verify_permission()
   return 0;
 }
 
+int RGWListBuckets::verify_permission(Jager_Tracer& tracer, const Span& parent_span)
+{
+  Span span = tracer.child_span("rgw_op.cc RGWListBuckets::verify_permission", parent_span);
+  rgw::Partition partition = rgw::Partition::aws;
```

Comment:
can you remove non-jaeger changes in the further cleanups, it would make work easier for reviewer, hence better focus on feedback.